src/components/manage/Blocks/HTML/Edit.jsx

```diff
@@ -9,15 +9,15 @@ import Editor from 'react-simple-code-editor';
 import { highlight, languages } from 'prismjs/components/prism-core';
 import 'prismjs/components/prism-markup';
 import { Button } from 'semantic-ui-react';
-import loadable from '@loadable/component';
 import { defineMessages, injectIntl } from 'react-intl';
 
 import { Icon } from '@plone/volto/components';
 import showSVG from '@plone/volto/icons/show.svg';
 import clearSVG from '@plone/volto/icons/clear.svg';
 import codeSVG from '@plone/volto/icons/code.svg';
-
-const Pretty = loadable.lib(() => import('pretty'));
+import prettierSVG from '@plone/volto/icons/prettier.svg';
+import prettier from 'prettier/standalone';
+import parserHtml from 'prettier/parser-html';
```

Comment:
can we use loadable again? There's no reason for this to be loaded with the main bundle.

Reply:
Done